src/components/user/ChangePassword.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import lockIcon from '../../assets/lock-icon.png'; // Path to lock icon asset
+import logoWithCircleBorder from '../../assets/logo_with_circle_border-removebg.png'; // Path to logo asset
+
+// Styles
+const pageStyles = () => ({
+  width: '100%',
+  minHeight: '100vh',
+  overflow: 'auto',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+});
+
+const backgroundStyles = () => ({
+  backgroundColor: 'rgba(91, 46, 72, 1)',
+  width: '100%',
+  minHeight: '1024px',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  position: 'relative',
+  overflow: 'hidden',
+});
+
+const logoStyles = () => ({
+  marginBottom: '20px',
+  height: '150px', // Example size, adjust as needed
+  width: 'auto',
+});
+
+const headerStyles = () => ({
+  textAlign: 'center',
+  fontFamily: '"Montserrat"',
+  fontSize: '65px',
+  lineHeight: 'normal',
+  color: 'rgb(255, 255, 255)',
+  margin: '20px 0',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+});
+
+const lockIconStyles = () => ({
+  width: '110px', // Adjust size as needed
+  height: 'auto',
+  marginBottom: '20px', // Space between icon and heading
+  filter: 'invert(100%)', 
+  marginTop: '50px' // Invert colors to make the icon appear white
+});
+
+
+const labelStyles = () => ({
+  color: 'rgba(255, 255, 255, 1)',
```

Comment:
this is just white no?

Reply:
yes it is, i'll adjust it to just say white